homepage/locales/vn.ts

```diff
@@ -0,0 +1,83 @@
+export default {
+  translation: {
+    common: {
+      openInBrowser: 'Mở trong trình duyệt',
+      downloadApp: 'Tải ứng dụng',
+      fileSystemBasedStorage: 'Lưu trữ trên file',
+      comingSoon: 'Sắp ra mắt',
+      webApp: 'Ứng dụng web',
+      desktopApp: 'Ứng dụng desktop',
+      mobileApp: 'Ứng dụng điện thoại',
+    },
+    header: {
+      forTeams: 'Dành cho teams',
```

Comment:
Not sure if we should change `teams` to `nhóm`

Reply:
nhóm/đội ngũ is pretty good (suggested by thai). Maybe should use that